### PR TITLE
Title shortening suggestions

### PR DESCRIPTION
**Pull request by TJaniF**
As per the Slack discussion :)

These titles are just what made sense to me, happy to hear other suggestions!

### CODE REVIEW

File: learn/use-case-airflow-dbt.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: "ELT pipelines with Airflow, dbt Core, Cosmos, and the Astro Python SDK"
+title: "ELT pipelines with Airflow and dbt Core"
```
Comment (kentdanas) [inline learn/use-case-airflow-dbt.md:2]: I think we should be consistent between this and the databricks one with either "ELT pipelines" or just "ELT". I would probably lean towards just "ELT", but not a strong preference.

File: learn/cloud-ide-tutorial.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: 'Write and schedule a simple ML pipeline using the Astro Cloud IDE'
-sidebar_label: 'Astro Cloud IDE - Write an ML pipeline'
+title: 'ML with the Astro Cloud IDE'
+sidebar_label: 'ML with the Astro Cloud IDE'
```
Comment (kentdanas) [inline learn/cloud-ide-tutorial.md:3]: Do you think ML Ops would make sense here? Or maybe "ML pipelines"? Just "ML" seems too broad to me, at least for the title. For the sidebar label I think it's fine.

**Review by kentdanas — APPROVED** [review]
Looks good! A few minor comments, but I'm okay with whatever you decide for these.

File: learn/cloud-ide-tutorial.md (hunk 2 above)
Comment (TJaniF) [reply learn/cloud-ide-tutorial.md:3]: I think ML Ops might give people false expectations, like interaction with ML Ops tools. I do like ML pipelines, will change it to that :)

**Review by TJaniF — COMMENTED** [review]
@kentdanas thank you for the review, used both suggestions :)

**Pull request merged**